packages/schematics/angular/component/files/__name@dasherize@if-flat__/__name@dasherize__.__type@dasherize__.spec.ts.template

```diff
@@ -7,7 +7,7 @@ describe('<%= classify(name) %><%= classify(type) %>', () => {
   let fixture: ComponentFixture<<%= classify(name) %><%= classify(type) %>>;
 
   beforeEach(async(() => {
```

Comment:
Can you turn this into an actual async function?  The await won’t work otherwise.  Please also remove the import of “async” as well if it is not used with the file anymore.

Reply:
Fark! I misread the async I thought it said:
```JavaScript
  beforeEach(async () => {
    TestBed.configureTestingModule({
      declarations: [ MyComponent ]
    })
    .compileComponents();
  });
```

I guess the async is a function call to some angular testing thingy :(

Reply:
Guess it's not a problem then... Will close.

Reply:
The async function was used prior to native async/await support in the test framework.
If you are still interested, it would still be a useful change.